### ISSUE TITLE
How to visualize time in x axis instead of steps?

### ISSUE THREAD

**Issue by JaslinRikkaz123** [projectmesa/mesa#1065]
**What's the problem this feature will solve?**
I need to visualize the time value that i have modeled in my  model.py in x-axis of chart module instead of steps.
How to do that suggest me a solution

**Comment by jackiekazil** [comment]
If you consider a step as a unit of time, then I think this is solved. Yes? No?

**Comment by JaslinRikkaz123** [comment]
No, I mean time should be in x axis like 8.30,8.35,8.45...something like
that
